BootloaderCommonPkg/Library/NvmExpressLib/NvmExpressHci.c

```diff
@@ -2,7 +2,7 @@
   NvmExpressDxe driver is used to manage non-volatile memory subsystem which follows
   NVM Express specification.
 
-  Copyright (c) 2013 - 2016, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2013 - 2020, Intel Corporation. All rights reserved.<BR>
   SPDX-License-Identifier: BSD-2-Clause-Patent

```

Comment:
we should not update its license year since we don't have code change in this file and NvmExpressBlockIo.h

Reply:
OK, I had thought I changed something there.  Maybe I was wrong.  :)
I will roll this back.